scripts/deployPriceFeed/deploy.js

```diff
@@ -0,0 +1,52 @@
+const { ethers, upgrades } = require('hardhat');
+const { toBytes32 } = require('../../index');
+const { setTargetAddress } = require('../helpers');
+
+async function main() {
+	let accounts = await ethers.getSigners();
+	let owner = accounts[0];
+	let networkObj = await ethers.provider.getNetwork();
+	let network = networkObj.name;
+	if (network === 'unknown') {
+		network = 'localhost';
+	}
+
+	if (network == 'homestead') {
+		network = 'mainnet';
+	}
+
+	if (networkObj.chainId == 69) {
+		networkObj.name = 'optimisticKovan';
+		network = 'optimisticKovan';
+	}
+	if (networkObj.chainId == 10) {
+		networkObj.name = 'optimistic';
+		network = 'optimistic';
+	}
+
+	console.log('Account is: ' + owner.address);
+	console.log('Network:' + network);
+	console.log('Network id:' + networkObj.chainId);
+
+	const PriceFeed = await ethers.getContractFactory('PriceFeed');
+	const priceFeed = await upgrades.deployProxy(PriceFeed, [owner.address]);
+	await priceFeed.deployed();
+
+	console.log('PriceFeed deployed to:', priceFeed.address);
+	setTargetAddress('PriceFeed', network, priceFeed.address);
+
+	const aggregators = require(`./aggregators/${network}.json`);
+	for (let [key, aggregator] of Object.entries(aggregators)) {
+		let tx = await priceFeed.addAggregator(toBytes32(key), aggregator);
+		await tx.wait().then(e => {
+			console.log('PriceFeed: addAggregator for', key);
+		});
+	}
```

Comment:
add this part also
https://discord.com/channels/@me/919973164203774002/921043308954939392